Resources/public/js/views/services/plugins/ez-objectrelationsloadplugin.js

```diff
@@ -25,7 +25,7 @@ YUI.add('ez-objectrelationsloadplugin', function (Y) {
         },
 
         /**
-         * Loads the related contents. Once this is done, it sets the contents in
+         * Loads the related contents. Once this is done, it sets the content structs in
```

Comment:
that's a BC break. This is also related to this comment https://github.com/ezsystems/PlatformUIBundle/pull/477/files#r47764778
We have 2 options here:
1. we keep the BC and when the event is fired without the `loadLocation` (nor the `loadLocationPath`) flag, we're still sending back only an array of Content models
2. we do this BC break, so we have to document it (I guess we need to create bc document in the repository ?)

Also, in both case, we need to document the new behaviour in the API doc so we have to add the `@since` tag, I guess with the value `2016.02` ?

I would choose 1. even if that's not super consistent in terms of API.
ping @andrerom @bdunogier